src/github_domain.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects for github entities."""
+
+from __future__ import annotations
+
+import datetime
+
+from typing import Any, Dict, List, Optional, Type
+
+DEFAULT_TIMESTAMP: datetime.datetime = (
+    datetime.datetime(datetime.MINYEAR, 1, 1, tzinfo=datetime.timezone.utc))
+
+
+class Assignee:
+    """A class representing an assignee of a pull request."""
+
+    def __init__(
+        self,
+        username: str,
+        timestamp: datetime.datetime = DEFAULT_TIMESTAMP
+    ) -> None:
+        """
+        Args:
+            username: str. GitHub username of the assignee.
+            timestamp: datetime. The time when an event happened.
+        """
+        self.username = username
+        self.timestamp = timestamp
+
+    def set_timestamp(self, timestamp: datetime.datetime) -> None:
+        """Sets timestamp to assignee.
+        Args:
```

Comment:
Add a newline above this one.

Reply:
done, `outdated` is not showing here because the method name is also changed.